test/specs/loginFormTestsTwo.js

```diff
@@ -0,0 +1,56 @@
+import { browser, $ } from '@wdio/globals';
+import { assert } from 'chai';
+
+
+describe('Validation of elements in the login form on https://www.saucedemo.com/', () => {
+
+    it('should check the elements of a registration form and that the default credentials are displayed', async () => {
+        await browser.url('https://www.saucedemo.com/');
+        const errorMessage = "Text doesn't match!"
+
+        // Check the username field’s placeholder/The placeholder is Username
+        const usernamePlaceholder = await $("//input[@placeholder='Username']")
+        const usernamePlaceholderText = await usernamePlaceholder.getAttribute('placeholder');
+        assert.equal(usernamePlaceholderText, "Username", errorMessage);
+
+        // Check that password fields is present/The placeholder is Password
+        const passwordPlaceholder = await $("//input[@placeholder='Password']")
+        const passwordPlaceholderText = await passwordPlaceholder.getAttribute('placeholder');
+        assert.equal(passwordPlaceholderText, "Password", errorMessage);
+
+        // Check the Login button/Text is Login. Color of the button is green (#3ddc91)
+        const loginButton = await $("//input[@data-test='login-button']");
+        const loginButtonText = await loginButton.getAttribute('value');
+        const elementColor = await loginButton.getCSSProperty('background-color');
+        const expectedColor = elementColor.parsed.hex;
+
+        assert.equal(loginButtonText, "Login", errorMessage)
+        assert.equal(expectedColor, '#3ddc91', "Color doesn't match");
```

Comment:
I'd recommend to create a constant for this, something like COLOR_GREEN and use it instead of hex code